disco_aws_automation/disco_acm.py

```diff
@@ -6,27 +6,45 @@
 import boto3
 import botocore
 
+CERT_SUMMARY_LIST_KEY = 'CertificateSummaryList'
+CERT_ARN_KEY = 'CertificateArn'
+DOMAIN_NAME_KEY = 'DomainName'
+
 
 class DiscoACM(object):
     """
     A class to manage the Amazon Certificate Service
 
     """
+    WILDCARD_PREFIX = "*."
 
     def __init__(self, connection=None):
         self._acm = connection
 
+    def _in_domain(self, domain, dns_name):
```

Comment:
You're right that would be simpler, but it seemed worth supporting wildcard domains below the top level just in case.
e.g. a.b.c.d  and *.c.d